config_panel.toml

```diff
@@ -0,0 +1,253 @@
+version = "1.0"
+
+[main]
+name = "Main Settings"
+services = ["matrix-__APP__"]
+
+    [main.welcome]
+    name = "Experience for new Users"
+
+        [main.welcome.enable_registration]
+        ask = "Enable Registration for new users."
+        type = "boolean"
+        yes = true
```

Comment:
```suggestion
        yes = "true"
```

Reply:
You may need quotes around `true` and `false`

Reply:
done